scripts/combine/setupCombine.py

```diff
@@ -72,6 +72,7 @@ def make_parser(parser=None):
     parser.add_argument("--effStatLumiScale", type=float, default=None, help="Rescale equivalent luminosity for efficiency stat uncertainty by this value (e.g. 10 means ten times more data from tag and probe)")
     parser.add_argument("--binnedScaleFactors", action='store_true', help="Use binned scale factors (different helpers and nuisances)")
     parser.add_argument("--isoEfficiencySmoothing", action='store_true', help="If isolation SF was derived from smooth efficiencies instead of direct smoothing")
+    parser.add_argument("--nonClosureUnc", action='store_true', help="Include uncertainty from muon momentum scale nonclosure")
```

Comment:
New option is not necessary, can just make the default "none" for --nonClosureScheme here

https://github.com/WMass/WRemnants/blob/87700ff6fa8568db710743e65e2dee9f45790911/utilities/common.py#L183